msm8937.mk

```diff
@@ -82,37 +82,48 @@ PRODUCT_PACKAGES += \
     libqcompostprocbundle
 
 PRODUCT_PACKAGES += \
-    android.hardware.audio@2.0-impl \
+    android.hardware.audio@4.0-impl \
     android.hardware.audio@2.0-service \
-    android.hardware.audio.effect@2.0-impl \
+    android.hardware.audio.effect@4.0-impl \
+    android.hardware.audio.effect@2.0-service \
```

Comment:
Why service 2.0?

Reply:
impl is 4.0, service is 2.0, idk why
but it's like that everywhere else, I'm assuming a 4.0-service doesn't exist